{
    "GSD": {
        "alias": "CVE-2021-20251",
        "id": "GSD-2021-20251",
        "references": [
            "https://www.suse.com/security/cve/CVE-2021-20251.html"
        ]
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "cve@mitre.org",
                "ID": "CVE-2021-20251",
                "STATE": "RESERVED"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2021-20251"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "A flaw was found in samba. A race condition in the password lockout code may lead to the risk of brute force attacks being successful if special conditions are met."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1929800",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1929800"
                        },
                        {
                            "name": "https://bugzilla.samba.org/show_bug.cgi?id=14611",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://bugzilla.samba.org/show_bug.cgi?id=14611"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2023-03-06T23:15Z",
            "publishedDate": "2023-03-06T23:15Z"
        }
    }
}